app/components/TablePaginationControls.tsx

```diff
@@ -47,8 +58,7 @@ export default function TablePaginationControls({
           }}
         >
           <AlignedLabel>
-            <FaCircleArrowLeft />
-            <span className="ms-2">Prev</span>
+            <FaAngleLeft />
```

Comment:
This is a response to some of the questions which you pose in your PR's top-line comment. I'm with you about not caring about how many pages there are generally, but I do care about where I am in relation to the book-ends of those pages. To wit, I would suggest we add some `<<` and `>>` buttons on the outside of the `<` and `>` ones that beam to you to the first or last page respectively. This gives me a way to get to the end of the set if I do happen to care about the number of pages, which does happen sometimes.

For example, here's a little non-fiction drawn from personal experience: If one gets the task of QA/QC'ing all the fatal and serious injury crashes in the year 2017, knowing how many pages you have left is really helpful.